snippets/fsharp/System/TupleT1,T2,T3,T4,T5,T6,T7,TRest/Item1/item1.fs

```diff
@@ -0,0 +1,67 @@
+﻿open System
```

Comment:
It looks like you forgot the `// <Snippet1>` tags in the various Item snippets. That's what caused the build warnings. @albert-du @BillWagner

Reply:
@albert-du Can you make an updated PR, or should I?

Reply:
oops looks like this one fell off my radar a bit, thank you for fixing it!